### ISSUE TITLE
Create Heads Up Display

### ISSUE THREAD

**Issue by janithajc** [IllusionBox/Light-Beam-Pee-Alpa#5]

**Comment by IllusionBox** [comment]
Yeah Nishan and I will come up with  cool layout for the HUD :) Ill handle the UI layers. The basic things we need for now is the depleating health bar which is connected to movement :)

**Comment by maleenj** [comment]
Yeah Nishan and I will come up with cool layout for the HUD :) Ill handle the UI layers. The basic things we need for now is the depleating health bar which is connected to movement :)
